exercises/book-store/src/test/java/BookStoreTest.java

```diff
@@ -0,0 +1,126 @@
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Test;
+
+public class BookStoreTest{
+
+	@Test
+	public void Only_a_single_book(){
```

Comment:
Standard convention in Java is to use `camelCase` for method and variable names.  Save the `snake_case` for Python!